### PR TITLE
Add npm-based changelog generator

### PR DESCRIPTION
**Pull request by maniax89**
I have a PR up to add in author URLs (similar to what is included with this PR)

https://github.com/lalitkapoor/github-changes/pull/61

Until it gets merged, we can use my fork:

https://github.com/maniax89/github-changes/tree/author_url

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +66,7 @@
     "start": "webpack-dev-server --env dev",
     "dist": "del dist; webpack --env prod",
     "test": "karma start",
-    "test:watch": "npm test -- --no-single-run --auto-watch"
+    "test:watch": "npm test -- --no-single-run --auto-watch",
+    "generate-changelog": "node node_modules/github-changes -o sapientglobalmarkets -r react-querybuilder -a --use-commit-body --only-pulls --date-format=\"(YYYY/MM/DD)\""
```
Comment (maniax89) [inline package.json:72]: not sure how to shorten this right now

**Comment by maniax89** [comment]
not sure how well this would work when you had tons of history though - seems to be an all-or-nothing generation (no option to append to existing changelog) but it is OK for now

**Comment by maniax89** [comment]
Addresses #16

**Comment by maniax89** [comment]
fixed merge conflicts

**Comment by maniax89** [comment]
any updates on this?

**Comment by pavanpodila** [comment]
Merged. I’ll make an NPM release later today. Thanks for all the great updates!

**Comment by pavanpodila** [comment]
Published v1.3 on NPM: https://www.npmjs.com/package/react-querybuilder

**Pull request merged**